framework/framework.go

```diff
@@ -353,6 +285,12 @@ func (f *Framework) ProcessEvents(ctx context.Context, deleteChan chan watch.Eve
 //     }
 //
 func ProcessUpdate(ctx context.Context, obj interface{}, resources []Resource) error {
+	// NOTE it does the same thing as ProcessCreate.
+	err := process(ctx, obj, false, resources)
+	return microerror.Mask(err)
+}
```

Comment:
Now all `Process*` functions are almost the same. They invoke `process` with different `deleted` argument.